schemas/apiDefinition.swagger.schema.json

```diff
@@ -1350,19 +1358,57 @@
           "$ref": "#/definitions/schema/properties/enum"
         },
         "additionalProperties": {
-          "$ref": "#/definitions/schema/properties/additionalProperties"
+          "anyOf": [
+            {
+              "$ref": "#/definitions/schema-dynamic-allowed"
+            },
+            {
+              "type": "boolean"
+            }
+          ],
+          "default": {}
         },
         "type": {
-          "$ref": "#/definitions/schema/properties/type"
+          "allOf": [
+            {
+              "$ref": "http://json-schema.org/draft-04/schema#/properties/type"
+            }
+          ],
+          "not": {
+            "description": "Note: VS Code seems to be allowing 'file' for the type but it shouldn't. Explicitly denying it here seems to produce the correct results though.",
+            "enum": [
+              "file"
+            ]
+          }
         },
         "items": {
-          "$ref": "#/definitions/schema/properties/items"
+          "anyOf": [
+            {
+              "$ref": "#/definitions/schema-dynamic-allowed"
+            },
+            {
+              "type": "array",
```

Comment:
When 'items' can be an array? Based on the specification it is Object.

Reply:
See the original schema. It is allowed to be an object OR an array. In the array case, it indicates the items are a fixed-size array.

Reply:
Ok, it can be an array based on Json Schema. However, it is not supported in Power Automate and won't be de-serialized. OpenApi 3.0 explicitly says "items - Value MUST be an object and not an array.".
Since we are using OpenApi v2.0 we can keep this schema to follow the specification, or remove arrays to help create swaggers which will work in our system.

Reply:
I will keep it for now as it's an OpenApi v2.0 feature (we don't support OpenApi 3.0). IIRC, Power Apps does have some logic to support this.